README.md

```diff
@@ -1 +1,65 @@
-# siddhi-io-mqtt
\ No newline at end of file
+siddhi-io-mqtt
+======================================
+
+The **siddhi-io-mqtt** is an extension to <a target="_blank" href="https://wso2.github.io/siddhi">Siddhi</a> mqtt source and sink implementation,that publish and receive events from mqtt broker.
+
+Find some useful links below:
+
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt">Source code</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/releases">Releases</a>
+* <a target="_blank" href="https://github.com/wso2-extensions/siddhi-io-mqtt/issues">Issue tracker</a>
+## Latest API Docs 
+
```

Comment:
Latest API doc should come here. Ask Chathurika.

Reply:
fixed in 3ba15c9